test/testdrive/catalog.td

```diff
@@ -100,10 +100,10 @@ name
 ----
 materialize
 
-> SELECT id, name FROM mz_databases
```

Comment:
It seems like testdrive is creating and dropping databases behind the scenes to clean up state. That means we have no way of really knowing what the ID is going to be, because it depends on the order of tests.